### PR TITLE
added usage and environment var for db

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
I have NO recollection of this :/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,3 +46,35 @@ REF_FOLDER = '/mnt/hds/proj/bioinfo/MICROBIAL/REFERENCES/'
 PICARD_JAR = '/mnt/hds/proj/bioinfo/MICROBIAL/picard-tools-1.141/picard.jar'
 JAVA = '/mnt/hds/proj/bioinfo/mip/modules/jre1.7.0_45/bin/java'
 ```
+
+###Environment variable'
+```
+MWGS_SQL_DATABASE_URI 'SQLAlchemy database URI pointing to the database'
```
Comment (ingkebil) [inline README.md:52]: if this is a bash env var, it should be in the format of `KEY=value`

File: README.md [hunk 2]
```diff
@@ -43,3 +46,35 @@ REF_FOLDER = '/mnt/hds/proj/bioinfo/MICROBIAL/REFERENCES/'
...
+```
+
+###Starting analyses
+
+
+You can use the CLI to start analyses in two different ways: either for a particular sample or for all samples contained in a project.
+
+
+$ mwgs start [path to sample dir]
```
Comment (ingkebil) [inline README.md:61]:
```
$ mwgs start [path to sample dir]
```

File: README.md [hunk 3]
```diff
@@ -43,3 +46,35 @@ REF_FOLDER = '/mnt/hds/proj/bioinfo/MICROBIAL/REFERENCES/'
...
+
+This will launch an analysis right away. Since most samples have relatively few sequenced reads it doesn't take more than 10-20 minutes to complete but it's still not advisable to run it on the login-node for other than testing.
+
+
+You will more likely start analyses for all samples in a project at a time:
+
+
+$ mwgs project [path to project dir]
```
Comment (ingkebil) [inline README.md:69]:
```
$ mwgs project [path to project dir]
```

File: README.md [hunk 4]
```diff
@@ -43,3 +46,35 @@ REF_FOLDER = '/mnt/hds/proj/bioinfo/MICROBIAL/REFERENCES/'
...
+This command will generate a SBATCH script and submit it as a job to a node using SLURM. It parallelizes the analyses using GNU parallel to make use of all available cores on the node. When an analysis finishes it will automatically upload the results to the database.
+
+
+###Uploading analysis results
+
+
+If you need to upload results to the database manually you can simply point to the "statistics.yml" output file.
+
+
+$ mwgs add [path to stats file]
```
Comment (ingkebil) [inline README.md:80]:
```
$ mwgs add [path to stats file]
```

**Review by ingkebil — CHANGES_REQUESTED** [review]

**Comment by patrikgrenfeldt** [comment]
@sylvinite do you have any idéa what this is about?

**Comment by patrikgrenfeldt** [comment]
I'm closing this PR since I don't recollect creating it and no one else claimed it so far

**Pull request closed without merging**